calico-cloud_versioned_docs/version-3.17/threat/web-application-firewall.mdx

```diff
@@ -67,253 +67,226 @@ Enabling WAF for certain system services may result in an undesired cluster stat
+1. Select a service.
+2. Click the HTTP tab.
 
+To simulate an attack to view the default warning in Service Graph, follow these steps:
 
-#### Add or edit the rule set
+1. Get the cluster/service URL.
+   - The external address of your cluster/service
+   - The cluster IP of your application's service (if testing within the cluster)
+2. Enter the following command to trigger a SQL injection attack.
 
-1. Create a directory, and download the core rules set files that you want to use.
+##### Example: SQL Injection attack
 
-```bash
-mkdir my-ruleset && cd my-ruleset
-curl -O https://raw.githubusercontent.com/coreruleset/coreruleset/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf
 ```
-2. Download bootstrap files.
-
-  Download the following required bootstrapping configuration files [Reference](https://medium.com/lightbaseio/web-application-firewall-in-go-feat-owasp-modsecurity-core-rule-set-3f97a26e3311) that are used by Tigera operator:
-    - `modsecdefault.conf`
-    - `crs-setup.conf`
-
-:::note
-
-The file names MUST be in all lowercase or they will fail to upload. They are loaded into ModSec before any REQUST-*.conf Core Rules Set files.
-
-:::
-
-```bash
-curl -O https://raw.githubusercontent.com/lsgroup/SmartReverseProxy/master/modsecdefault.conf
-curl https://raw.githubusercontent.com/coreruleset/coreruleset/v3.3/dev/crs-setup.conf.example > crs-setup.conf
+  curl http://my.loadbalancer.address/cart?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user
 ```
 
-3. Change your current directory to the my-ruleset folder where your core rules set files live. Create a configMap containing all the files downloaded into your new directory and replace the existing rule set with it:
-
+In Service Graph, go to the service and click the HTTP tab. You will see the following warning:
 
-```bash
-kubectl create cm --dry-run=client --from-file=. -o yaml -n tigera-operator modsecurity-ruleset > ../my-ruleset.yaml
-kubectl replace -f ../my-ruleset.yaml
 ```
+  level=warning msg="WAF Process Http Request [2f435aca-4a3e-4f96-a7b2-a60c9745bd7b] URL '/test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user' OWASP Warning'[2] Host:'10.224.0.38' File:'/etc/modsecurity-ruleset/REQUEST-942-APPLICATION-ATTACK-SQLI.conf' Line:'45' ID:'942100' Data:'' Severity:'0' Version:'OWASP_CRS/3.3.2' Message:'Warning. detected SQLi using libinjection.''"
+```  
 
-### Enable WAF using CLIs
+#### Threat defense
 
-1. Create or update the [ApplicationLayer](../reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` field, and ensure it is set to `Enabled`.
+To view WAF events in a centralized security events dashboard, go to: **Threat defense**, **Security Events**. For help, see [Security Event Management](./security-event-management).
```

Comment:
Is Security Event Management new in v3.18?

Reply:
Yes.